<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>author:xtt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_file">
        <property name="text">
         <string>file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelW">
        <property name="text">
         <string>width:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelH">
        <property name="text">
         <string>height:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_fps">
        <property name="text">
         <string>frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_av">
        <property name="text">
         <string>avfps</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPre">
        <property name="text">
         <string>pre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBreak">
        <property name="text">
         <string>break</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="imagelable">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
